app/settings.json

```diff
@@ -0,0 +1 @@
+{"Fuel_price": 2.4}
```

Comment:
what is it constant?

Reply:
why not? Should i use only uppercase?

Reply:
it's not correct, not setting.json, create a python constant and python module